src/start_env/mod.rs

```diff
@@ -293,7 +293,7 @@ fn start_ethereum_node(
                 node.node_image
                     .fund(
                         ethereum::derive_address(key),
-                        U256::from("9000000000000000000"),
+                        U256::from("7CE66C50E2840000"), // 9_000_000_000_000_000_000: 9 Ethers
```

Comment:
:question: How about [formatting a `u128` to hex](https://doc.rust-lang.org/std/fmt/trait.LowerHex.html)?